dask/dataframe/io/tests/test_io.py

```diff
@@ -221,6 +221,7 @@ def test_from_bcolz_column_order():
 
 def test_from_pandas_dataframe():
     a = list("aaaaaaabbbbbbbbccccccc")
+    np.random.seed(123)
```

Comment:
This should be testing behavior that is independent of the values in the `b` column. Did you find that not having the seed set resulting in this test failing?

Reply:
Yea, agreed that it shouldn't matter here. I added this a while back unfortunately and don't remember the specifics.

My guess is I was debugging general `from_pandas` failures with my iloc implementation, and found it hard to follow when stepping repeatedly in the debugger when the values weren't the same each time.

Happy to remove if it feels like a distraction (or, to file an issue for a more holistic audit that all random tests are seeded, which I feel like should be the default in general)